mindjunkies/forums/models.py

```
@@ -34,76 +39,76 @@ def save(self, *args, **kwargs):
 
+class LikedComment(models.Model):
+    comment = models.ForeignKey(ForumComment, on_delete=models.CASCADE)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+    created = models.DateTimeField(auto_now_add=True)
+    
+    def __str__(self):
+        return f'{self.user.username} : {self.comment.body[:30]}'
+    
+class Reply(models.Model):
+    author = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.SET_NULL,null=True, related_name="replies")
+    parent_comment = models.ForeignKey(ForumComment, on_delete=models.CASCADE, related_name="replies",null=True,blank=True)
```

Ruff (0.8.2)

Line too long (123 > 119)

(E501)